.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-  - "2.6"
   - "2.7"
   - "3.3"
   - "3.4"
```

Comment:
Could you add python 3.5 and 3.6 to Travis?

Reply:
it's already there!

Reply:
D'oh! Thanks, I missed it.